addons/docs/docs/recipes.md

```diff
@@ -235,6 +236,37 @@ addParameters({
 });
 ```
 
+## Customizing source snippets
+
+As of SB 6.0, there are two ways to customize how Docs renders source code, via story parameter or via a formatting function.
+
+If you override the `storySource.source` parameter, the `Source` block will render whatever string is added:
+
+```js
+const Example = () => <Button />;
+Example.story = {
+  parameters: {
+    storySource: { source: 'some arbitrary string' },
```

Comment:
Would you not set `docs.source`?

Reply:
Ah, left over from the old code. Will fix.